### PR TITLE
corrections in grammar

### PR DESCRIPTION
**Pull request by Nabela**
Afteer seeing this online, I have noticed some grammar issues, so I corrected them. Now it finaly should be ok. :)

### CODE REVIEW

File: countries.json [hunk 1]
```diff
@@ -6610,7 +6610,7 @@
 			"por": {"official": "saint Martin", "common": "S\u00e3o Martinho"},
 			"rus": {"official": "\u0421\u0435\u043d-\u041c\u0430\u0440\u0442\u0435\u043d", "common": "\u0421\u0435\u043d-\u041c\u0430\u0440\u0442\u0435\u043d"},
 			"spa": {"official": "Saint Martin", "common": "Saint Martin"},
-			"svk": {"official": "Sv\u00e4t\u00fd Martin", "common": "Sv\u00e4t\u00fd Martin"},
+			"svk": {"official": "Sv\u00e4t\u00fd Martin", "common": "Sv\u00e4t\u00fd Martin, FR"},
```
Comment (mledoze) [inline countries.json]: @Nabela, why `, FR` was added here?

Comment (herrjemand) [inline countries.json]: @Nabela https://sk.wikipedia.org/wiki/Sint_Maarten

Comment (herrjemand) [inline countries.json]: @Nabela so FR should be removed.

Comment (Nabela) [inline countries.json]: like i said: ".in slovak it has same name" and it is Svätý Martin for both parts of country. so FR should stay there i suppose.

Comment (herrjemand) [inline countries.json]: @Nabela "Svätý Martin" is a name of an island, however to the rules of the Slovak language, personal names do not translate. So there should be Sint-Maarten and Saint-Marten

**Comment by mledoze** [comment]
Thank you @Nabela

**Pull request merged**